### DISCUSSION TITLE
250

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#611] [Comments]
http://jojoldu.tistory.com/250

**Comment by jojoldu** [comment]
작성자: 기스하워드

크리스미스 선물같은 포스트 고맙습니다! 전 인텔리j 반년 정도 쓰고 있는데, 터미널에서 init을 한다든가 소스트리로 깃을 관리한다던가 하는 것은 생각을 못 해봤어요. 모든 것을 인텔리j의 메뉴 안에서 해결해야 하는 줄... ㅜ 이후 글들도 기대됩니다. 즐거운 성탄되세요~

**Comment by jojoldu** [comment]
작성자: Mr.star

제가 좋아하는 Blow-by-blow 스타일의 자세한 설명과 스크린샷 감사합니다. 다음편이 기대됩니다! (요즘 저는 블로그 포스팅을 이런 저런 핑계로 못하고 있는데 개인적인 일이 어느정도 정리되면 동욱님처럼 다른 사람과 스스로에게 도움이 되는 양질의 글을 많이 작성하도록 하겠습니다.)

**Comment by jojoldu** [comment]
작성자: 임승현

처음 학교에서 배우지도 않은 스프링부트로 홈페이지를 만든다고 했을때, 구글에 스프링부트를 검색했으나 나오는자료들은 일반 스프링프레임워크 였고, 심지어 스프링과 스프링부트의 차이점조차 한글문서로 찾을수가 없었습니다. 결국 하나하나 삽질할수밖에 없었습니다. (모델2, mybatis, 어노테이션,jstl , 의존성...) 이런 좋은강의를 제공해주셔서 정말 감사합니다.

**Comment by jojoldu** [comment]
작성자: soulduse@naver.com

.gitignore 복붙하면 티스토리 특성상(?) 
정렬이 제대로 되지 않아서 다른분들을 위해 기록합니다 :)


/build/ 
!gradle/wrapper/gradle-wrapper.jar 
/out 

### STS ### 
.apt_generated 
.classpath 
.factorypath 
.project 
.settings 
.springBeans 

### IntelliJ IDEA ### 
.idea 
*.iws 
*.iml 
*.ipr 

### NetBeans ### 
nbproject/private/ 
build/ 
nbbuild/ 
dist/ 
nbdist/ 
.nb-gradle/

**Comment by jojoldu** [comment]
작성자: soulduse@naver.com

너무 좋은 글이네요! 
쉽게 설명되어있어서 이해가 쏙쏙 되었습니다!
너무 고마워서 광고를 누르고싶은데 광고가 안보이네요

**Comment by jojoldu** [comment]
작성자: 잘 보고있습니다.

친절한 설명 감사합니다!

**Comment by pds911** [comment]
:) 좋은 글 감사합니다!

**Comment by jojoldu** [comment]
@pds911 댓글 감사합니다^^

**Comment by deepweller** [comment]
좋은 글 감사합니다. 설명이 정말 자세하네요!

**Comment by inhyuck** [comment]
jojoldu님 블로그를 토대로 intellij IDE를 이용한 간단한 spring project를 시작하려 합니다. 감사합니다 :D

**Comment by jojoldu** [comment]
@deepweller 고맙습니다 :)

**Comment by jojoldu** [comment]
@inhyuck 축하드립니다! IntelliJ 정말 좋아요! ㅋㅋ

**Comment by hcyfor** [comment]
안녕하세요 향로님!
유투브에서 자바지기님의 spring boot 강의를 듣다가 재밌어서 이것저것 검색하다가, 향로님의 게시글까지 오게 되었습니다. 너무 디테일하게, 친절하게 적어주셔서 정말 감사하다는 말씀드립니다.
현재 IT 쪽의 기업들이 어떤 기술들을 쓰고 있는지까지 언급해주셔서 알차게 읽고 있습니다!ㅎㅎ

**Comment by jojoldu** [comment]
@hcyfor 앗 칭찬 감사합니다 :)
다음에 좀 더 자세하고, 실전에 가까운 내용으로 정리해서 공유드리겠습니다.

**Comment by yujineeee** [comment]
크...! 도움이 정말 많이 되었습니다ㅠㅠ 매번 spring project 시작하려다 실패 했었는데 첨으로 발을 내딛게 되었네요☺️ 좋은 글 감사합니다!

**Comment by alittlenation** [comment]
좋은 강의 감사합니다.

**Comment by alittlenation** [comment]
근데 commit만 했는데 올라가네요? push도 해야하는거 아닌가염 깃헙레포에 올라가려면

**Comment by DUHEON** [comment]
안녕하세요
SpringBoot를 이용해서 webService를 시작하고 싶었는데
jojoldu님의 블로그를 발견하게 됐습니다.

저는 개발툴을 vsCode를 사용하여
블로그 내용을 접목시켜보려고 하는데요
이렇게 작성한 내용을 출처를 표시하여 제 블로그에 작성해도 괜찮을런지요?

**Comment by DaehunGwak** [comment]
좋은 글 감사합니다!! :+1:

**Comment by jaewon0913** [comment]
안녕하세요.
좋은 강의 감사합니다.
저는 현재 SpringBoot 2.2.0 Version으로 강의를 따라가고 있는데

프로젝트 생성 부분부터
Developer Tools -> Lombok / Web -> Spring Web / SQL -> Spring Data JPA , H2 Database / Ops -> Spring boot Actuator 로 생성을 하였고

2번째 강의에서 테스트코드 작성중 @Runwith 와 @After , assertThat이 임포트가 안되는 문제가 발생하였는데
뭐가 문제일까요..? 

현재 저의 build.gradle 입니다.
plugins {
    id 'org.springframework.boot' version '2.2.0.RELEASE'
    id 'io.spring.dependency-management' version '1.0.8.RELEASE'
    id 'java'
}

group = 'com.springboot'
version = '0.0.1-SNAPSHOT'
sourceCompatibility = '1.8'

configurations {
    developmentOnly
    runtimeClasspath {
        extendsFrom developmentOnly
    }
    compileOnly {
        extendsFrom annotationProcessor
    }
}

repositories {
    mavenCentral()
}

dependencies {
    implementation 'org.springframework.boot:spring-boot-starter-actuator'
    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
    implementation 'org.springframework.boot:spring-boot-starter-web'
    compileOnly 'org.projectlombok:lombok'
    developmentOnly 'org.springframework.boot:spring-boot-devtools'
    runtimeOnly 'com.h2database:h2'
    annotationProcessor 'org.projectlombok:lombok'
    testImplementation('org.springframework.boot:spring-boot-starter-test') {
        exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
    }
}

test {
    useJUnitPlatform()
}

**Comment by jaewon0913** [comment]
댓글이 지워지지 않아 다시 남깁니다..
jUnit5 버전으로 올라오면서 jUnit4와 다른 것을 확인하였고 , 현재 공식문서를 통해서 해결하고있습니다.
좋은 강의 감사합니다~~~

**Comment by chanchancheers** [comment]
복받으세요
